blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -227,6 +327,26 @@ exports.fetchLogsInjectStart = function fetchLogsInjectStart(url, start, onSucce
             }
         });
 };
+
+/**
+ * Determines if the provided object is a "run" type
+ */
+function isRun(obj) {
+    return obj._class === 'io.jenkins.blueocean.rest.impl.pipeline.PipelineRunImpl'
```

Comment:
This I must have run synchronously, I can't really adapt it to an async pattern, and I don't have an async hook to load up capabilities in this case, no react component, etc... need to revisit this and capabilities as a whole, as we've discussed